.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberParser.cs

```diff
@@ -13,9 +13,9 @@ public class BaseNumberParser : IParser
             new Regex(@"\d+", RegexOptions.Singleline);
 
         private static readonly Regex MultiDecimalSeparatorCultureRegex =
-            new Regex(@"^(en|es|fr)(-)?\b", RegexOptions.Singleline | RegexOptions.Compiled);
+            new Regex(@"^(en|es|fr|bg)(-)?\b", RegexOptions.Singleline | RegexOptions.Compiled);

```

Comment:
Is Bulgarian really multi-decimal separator? For example, are both 5.0 and 5,0 the same number?

Reply:
Thank you @tellarin,
Yes, now days "." and "," are interchangeable as a decimal separator. The formal rule is "," but a decimal point is increasingly used (influenced by electronic devices and internet).
In addition, in regular expressions I've also included variants of common misspelling in some words.